src/views/Home/components/Prizes/prizes.scss

```diff
@@ -1,3 +1,87 @@
-/* stylelint-disable-line no-empty-source */
+@use "~stylesheets/typography";
+@use "~stylesheets/responsiveness";
 
-/* Delete these lines when you begin development */
+.Prizes {
+  position: relative;
+  margin: 0;
+
+  &__background {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    z-index: 0;
+  }
+
+  &__container {
+    position: relative;
+    max-width: 2000px;
+    min-height: 500px;
+    margin: 0 auto;
+  }
+
+  &__title {
+    @include typography.anaglyph-title-text;
+
+    padding: var(--extra-extra-large-spacer) 0 var(--medium-spacer) 0;
+    text-align: center;
+  }
+
+  &__boxContainer {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    max-width: 1500px;
+    margin: var(--medium-spacer) auto;
+  }
+
+  &__box {
```

Comment:
@tim-nguyen-cs Let me know how this looks: 

<img src="https://user-images.githubusercontent.com/20177171/102257278-64760b00-3ec1-11eb-94bc-c46b8f2c857f.png" width="500px" />

I'm not enforcing a height here because it has to be rectangular on smaller displays to fit everything, but I think it is squarish on most of the screen sizes now.

Reply:
Looks great!